apps/hexagon_benchmarks/conv3x3_generator.cpp

```diff
@@ -48,6 +51,12 @@ class Conv3x3 : public Generator<Conv3x3> {
                 .tile(x, y, xi, yi, vector_size, 4, TailStrategy::RoundUp)
                 .vectorize(xi)
                 .unroll(yi);
+            if (use_prefetch_sched) {
+                output.prefetch(input, y, 2);
+            }
+            if (use_parallel_sched) {
+                output.parallel(y);
```

Comment:
BTW, HelloHexagon implements this strategy for line buffering: https://github.com/halide/Halide/blob/master/apps/HelloHexagon/blur_generator.cpp#L56

The difference is HelloHexagon just splits into 128 line chunks, rather than tries to split the image into 4 chunks. Either way works.

Reply:
Thanks Dillon, 
After incorporating your suggestion it seems to be working for me, and producing better performance. Look better?


            if (use_prefetch_sched) {
                output.prefetch(input, y, 2);
            }
            if (use_parallel_sched) {
                Var yo;
                output.split(y, yo, y, 128).parallel(yo);
            }

Reply:
Think i will try 4 big chunks now

Reply:
Looks better, when you write it this way, it actually is fine to just use output.parallel(y, 128). That's equivalent to what you wrote.

What you wrote is useful when you need to use 'yo' for something (e.g. if you were line buffering, you would need it to be able to store at the outer chunk).

Note that all of these benchmarks need this fix, not just this one.

Reply:
yep ack on all 5 files need the fix, thanks.

Reply:
ok if  i keep it as the split form?